practice-one/src/styles/sections/introduction.css

```diff
@@ -105,3 +105,77 @@
   vertical-align: middle;
   cursor: pointer;
 }
+
+/* Style for desktop screen under 1300px*/
+@media (max-width: 1300px) {
+  .introduction-container {
+    gap: 0px;
+    align-items: center;
+  }
+
+  .header-introduction {
+    font-size: var(--font-size-sub-header);
+  }
+
+  .para-introduction {
+    font-size: var(--font-size-xnormal);
+  }
+
+  .search-white-icon {
+    left: 88%;
```

Comment:
hack code

Reply:
What should i do in this case ? please give me advice :((